spec/models/collection_spec.rb

```diff
@@ -39,10 +39,20 @@
     end
   end
 
-  describe "when a single article in collection is updated" do
+  describe "#touch_articles" do
     it "touches all articles in the collection" do
-      random_article = collection.articles.sample
-      expect { random_article.touch }.to(change { collection.articles.map(&:updated_at) })
+      Timecop.freeze(DateTime.parse("2019/10/24"))
```

Comment:
This should be in a block:

```ruby
Timecop.freeze() do
end
```

so that the notion of `Time.current` gets reverted at the end of the test, otherwise it will affect other tests.

Reply:
Thanks for taking the time to review this @rhymes! I've addressed this comment in d21ddca and am now ready for another review 🙇